test/client_test.js

```diff
@@ -319,21 +333,38 @@ const connectionTests = connectionString => {
       await client.stop();
     }
   });
-};
 
-suite('Client', function() {
-  suite('constructor', async function() {
-    const monitor = await libMonitor({project: 'tests', mock: true});
-    test('rejects connectionString *and* username', function() {
-      assume(() => new Client({username: 'me', connectionString: 'amqps://..', monitor}))
-        .throws(/along with/);
-    });
-    test('requires either connectionString *or* username', function() {
-      assume(() => new Client({monitor}))
-        .throws(/is required/);
-    });
+  test('reclaim', async function() {
```

Comment:
This is testing `Client` functionality, so it belongs in the `Client` suite, below.